src/cryptenroll/cryptenroll-pkcs11.c

```diff
@@ -1,6 +1,13 @@
-/* SPDX-License-Identifier: LGPL-2.1-or-later */
+/* SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
please keep this line as is, we want the first line to be in exactly one form, so that we can easily process it automatically. i.e. open a separate comment below.